.gitignore

```
@@ -0,0 +1,8 @@
+book
+index.html
+
+# CIPs
+cips/*.html
+
+# WGs
+cips/wgs/*/*.html
```

should this have two asterisks to match any nested subdirectories?

```suggestion
cips/wgs/**/*.html
```